ios/FluentUI/People Picker/AvatarViewVnext.swift

```diff
@@ -308,66 +317,79 @@ public struct AvatarVnextView: View {
-                                        .frame(width: ringInnerGapSize, height: ringInnerGapSize, alignment: .center)
-                                        .overlay(Circle()
-                                                    .foregroundColor(Color(backgroundColor))
-                                                    .frame(width: avatarImageSize, height: avatarImageSize, alignment: .center)
-                                                    .overlay(avatarContent
-                                                                .frame(width: avatarImageSize * avatarImageSizeRatio, height: avatarImageSize * avatarImageSizeRatio, alignment: .center)
-                                                                .clipShape(Circle()),
-                                                             alignment: .center)
-                                        )
-                            )
-                         ,
-                         alignment: .center)
-                .mask(circularCutoutMask(targetFrameRect: CGRect(x: 0,
-                                                                 y: 0,
-                                                                 width: ringOuterGapSize,
-                                                                 height: ringOuterGapSize),
-                                         cutoutFrameRect: CGRect(x: presenceCutoutOriginCoordinates,
-                                                                 y: presenceCutoutOriginCoordinates,
-                                                                 width: presenceIconOutlineSize,
-                                                                 height: presenceIconOutlineSize))
-                        .fill(style: FillStyle(eoFill: shouldDisplayPresence)))
-                .overlay(shouldDisplayPresence ?
-                            AnyView(Circle()
-                                        .foregroundColor(isTransparent ? Color.clear : Color(tokens.ringGapColor))
-                                        .frame(width: presenceIconOutlineSize, height: presenceIconOutlineSize, alignment: .center)
-                                        .overlay(presence.image(isOutOfOffice: isOutOfOffice)
-                                                    .resizable()
-                                                    .frame(width: presenceIconSize, height: presenceIconSize, alignment: .center)
-                                                    .foregroundColor(presence.color(isOutOfOffice: isOutOfOffice)))
-                                        .frame(width: presenceIconFrameSideRelativeToOuterRing, height: presenceIconFrameSideRelativeToOuterRing, alignment: .bottomTrailing)
-                            )
-                            :
-                            AnyView(EmptyView()),
-                         alignment: .topLeading)
-        }
+        let bodyView = tokens.style == .group ?
+        AnyView(avatarContent
```

Comment:
I noticed that avatarContent was type-erased in its definition, does it need to be type-erased again here?

Reply:
Because bodyView is being assigned with one of the options returned by the ternary operator, they do need to be of the same type (avatarContent will be either an Image or Text vs the other option of the ternary operator that will be a Circle).

That's how I can assign it to a single bodyView variable and apply the onAppear modifier on it.